src/components/common/RadioButton/RadioButton.styled.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+interface StyledRadioButtonLabelProps {
+  disabled: boolean;
+}
+
+export const RadioButtonWrapper = styled.label<StyledRadioButtonLabelProps>`
+  ${({ disabled }) => css`
+    ${disabled && 'pointer-events: none; opacity: 0.5;'}
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    width: fit-content;
+    height: 1.2rem;
+    padding: 0 0;
+    cursor: pointer;
+    user-select: none;
+  `}
+`;
+
+export const RadioButtonInput = styled.input`
+  position: absolute;
+  width: 0;
+  height: 0;
+  opacity: 0;
+`;
```

Comment:
숨김요소를 써주실때 접근성 이슈가 생길 수 있을것 같은데 아래와 같이 사용해주는건 어떠실까요? 관련 레퍼런스 주소 남기겠습니다 :)
https://mulder21c.github.io/2019/03/22/screen-hide-text/
```suggestion
export const RadioButtonInput = styled.input`
 .s_a11yHidden {
  clip: rect(0 0 0 0);
  clip-path: polygon(0 0, 0 0, 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}
`;
```
스타일 순서는 저때는 stylelint를 abc순으로 먹여놔서 저 모양이에요....하하...

Reply:
chakra UI는 진짜 리액트용으로 데브툴도 고려해서 displayName까지 먹여주네요ㄷㄷ 역시 라이브러리

Reply:
#33 추가했습니다~